### DISCUSSION TITLE
v0.1.0

### DISCUSSION THREAD

**Discussion by AzulGarza** [Nixtla/neuralforecast#252] [Announcements]
## What's Changed
* Added Loss Function & Rewrote Unit Testing by @shibzhou in https://github.com/Nixtla/neuralforecast/pull/238
* fix reshapes and rnn by @cchallu in https://github.com/Nixtla/neuralforecast/pull/247
* mqnhits by @cchallu in https://github.com/Nixtla/neuralforecast/pull/248
* y to device by @cchallu in https://github.com/Nixtla/neuralforecast/pull/249
* Update LICENSE to MIT by @FedericoGarza in https://github.com/Nixtla/neuralforecast/pull/251

## New Contributors
* @shibzhou made their first contribution in https://github.com/Nixtla/neuralforecast/pull/238

**Full Changelog**: https://github.com/Nixtla/neuralforecast/compare/v0.0.9...v0.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Nixtla/neuralforecast/releases/tag/v0.1.0'>v0.1.0</a>.</em>

**Comment by gdevos010** [comment]
Thank you for making it an MIT license
